exercises/nth-prime/nth_prime_test.rb

```diff
@@ -22,13 +25,35 @@ def test_sixth_prime
 
   def test_big_prime
     skip
-    assert_equal 104_743, Prime.nth(10_001)
+    assert_equal 104743, Prime.nth(10001)
```

Comment:
If you wanted to be nice, you could make a helper method that formatted the numbers from the json into underscored versions in the test file.

(This is not at all a requirement, just an optional extra if you feel like it.)